master/getting-started/docker/tutorials/security-using-docker-labels-and-calico-policy.md

```diff
@@ -0,0 +1,186 @@
+to the workload endpoint.
+
+We have decided to use the label `role` indicating the role and a value of
+either `frontend` or `database`.  
+
+```
+docker run --label org.projectcalico.label.role=frontend --net net1 --name frontend-A -tid busybox
+docker run --label org.projectcalico.label.role=database --net net1 --name database-A -tid busybox
+```
+
+#### 3. Create policy
+
+Create the global policy to provide the required network isolation.
+
+Policy resources are defined globally, and include a set of ingress and egress
+rules and actions, where each rule can filter packets based on a variety
+of source or destination attributes (which includes selector based filtering
+using label selection).
+
+Each policy resource also has a "main" selector that is used to determine which
+endpoints the policy is applied to based on the applied labels.
+
+We can use `calicoctl create` to create two new policies for this:
+
+```
+cat << EOF | calicoctl create -f -
+- apiVersion: v1
+  kind: policy
+  metadata:
+    name: database
+  spec:
+    order: 0
+    selector: role == 'database'
+    ingress:
+    ingress:
+    - action: allow
+      protocol: tcp
+      source:
+        selector: role == 'frontend'
+      destination:
+        ports:
+        -  3306
+    - action: allow
+      source:
+        selector: role == 'database'
+    egress:
+    - action: allow
+      destination:
+        selector: role == 'database'
+- apiVersion: v1
+  kind: policy
+  metadata:
+    name: frontend
+  spec:
+    order: 0
+    selector: role == 'frontend'
+    egress:
+    - action: allow
+      protocol: tcp
+      destination:
+        selector: role == 'database'
+        ports:
+        -  3306
+EOF
+```
+
+This works as follows:
+
+-  Each database container is given the label `role = database`.
+-  Each frontend container in given the label `role = frontend`.
+-  The global policy resource "database" uses the selector `role == database` to
+   select containers with label `role = database` and applies ingress and egress
+   policy:
+   -  An ingress rule to allow TCP traffic to port 3306 from endpoints that have
+      the label `role = frontend` (i.e. from frontend containers since they are
+      the only ones with the label `role = frontend`)
+   -  An ingress and egress rule to allow all traffic from and to endpoints that
+      have the label `role = database` (i.e. from database containers).
+-  The global policy resource "frontend" uses the selector `role == frontend` to
+   select containers with label `role = frontend` and applies a single egress
+   rule to allow all TCP traffic to port 3306 on endpoints that have the label
+   `role = database` (i.e. to database containers)
+
+For details on all of the possible match criteria, see the
+[policy resource]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/profile)
+documentation.
+
+## Multiple networks
+
+Whilst the Docker API supports the ability to attach a container to multiple
```

Comment:
Maybe we can try to reword this to make it sound more positive?

?While some network providers tend to use different networks to enforce isolation, Calico philosophy opts to put all containers in the same network, and separate them using Calico policy. For this reason, Calico does not support attaching a container to multiple docker networks.